<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionWindow</class>
 <widget class="QDialog" name="SelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>263</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Select Visual Display</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SelectionWindow{
background-color: rgb(255,255,255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>TEXT BASED (D1)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="textBasedSelectAllCheckBox">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="textBasedBestAlternativeCheckBox">
           <property name="text">
            <string>Best alternative (V1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="textBasedRankedListCheckBox">
           <property name="text">
            <string>Ranked list (V2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="textBasedBestDerivedPropertyCheckBox">
           <property name="text">
            <string>Derived priorities (V3)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>GRAPHIC BASED (D2)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="graphicBasedSelectAllCheckBox">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="graphicBasedBestAlternativeCheckBox">
           <property name="text">
            <string>Best alternative (V1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="graphicBasedRankedListCheckBox">
           <property name="text">
            <string>Ranked list (V2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="graphicBasedBestDerivedPropertyCheckBox">
           <property name="text">
            <string>Derived priorities (V3)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>357</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generatePushButton">
        <property name="text">
         <string>generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
